charts/oes/templates/oesdb-deployment.yaml

```diff
@@ -14,20 +19,32 @@ spec:
       imagePullSecrets:
         - name: {{ .Values.k8sSecret}}
       containers:
-      - image: {{ .Values.OESDBImage }}
-        imagePullPolicy: IfNotPresent
+      - image: {{ .Values.opsmxdb.image.repository }}:{{ .Values.opsmxdb.image.tag }}
+        imagePullPolicy: {{ .Values.opsmxdb.image.pullPolicy }}
         name: db-opsmx
         ports:
         - containerPort: 5432
           protocol: TCP
-      {{- if .Values.persistence.enabled }}
```

Comment:
This should be an optional flag that a user can turn off if at all needed. This is seen in minio and redis charts and we should do the same.

Reply:
Now as we have moved to stateful set instead of the deployment, this persistence case no longer exists.

As a whole we have added a condition, that only if the user wants to use opsmx database then he can set the flag true in values.yaml.